themes/instrinsic/img/icons/web-badges/Readme.md

```diff
@@ -0,0 +1,114 @@
+    <img src="publicdomain.svg" width="81" height="21"
+      style="border-style: none;" alt="Public Domain Intrinsic Web Badge" />
+  </a>
+</p>
+
+<p xmlns:dct="http://purl.org/dc/terms/">
+This work (<span property="dct:title">Public Domain Intrinsic Web
+Badge</span>, by <a href="https://open.inf.is/DerekNonGeneric" rel="dct:creator">
+<span property="dct:title">Derek Lewis</span></a>), incorporates a derivative of
+<span property="dct:title"><code>File:PD-icon.svg</code></span>, by
+<a href="https://commons.wikimedia.org/wiki/File:PD-icon.svg" rel="dct:creator">
+<span property="dct:title">various authors</span></a>, identified by
+<a href="https://open.inf.is/DerekNonGeneric" rel="dct:publisher"><span
+property="dct:title">Derek Lewis</span></a>, is free of known copyright
+restrictions. “Public Domain Intrinsic Web Badge” is likewise placed into the
+Public Domain by Derek Lewis as he has dedicated all copyright interest in it
+to the Public Domain.</p>
+
+---
+
+## Valid Web Standards
+
+The “valid” web page & other web standard badges are used on documents that
+successfully passed validation for a specific technology. The W3C validation
+services image is used as a link to re-validate the document to show readers
+that one has taken some care to create an interoperable Web page.
+
+---
+
+### Valid HTML5 Intrinsic Web Badge
+
+```html
+<p>
+  <a href="https://validator.w3.org/check?uri=referer">
+    <img src="https://open.inf.is/assets/img/svg/valid-html5.svg?sanitize=true" alt="Valid HTML5!" height="21" width="81" />
+  </a>
+</p>
+
+This work, “Valid HTML5 Intrinsic Web Badge”, incorporates a derivative of
+<a href="https://www.w3.org/html/logo/#downloads">HTML5 Logo</a> by
+<a href="http://www.w3.org/">
+<abbr title="World Wide Web Consortium">W3C</abbr></a>, used under
+This work, “Valid HTML5 Intrinsic Web Badge”, incorporates a derivative of
+<a href="https://www.w3.org/html/logo/#downloads">HTML5 Logo</a> by
+<a href="http://www.w3.org/">
+<abbr title="World Wide Web Consortium">W3C</abbr></a>, used under
+<a rel="nofollow" class="external text"
+href="https://creativecommons.org/licenses/by/3.0/">CC BY</a> / Changed color,
+resized, and optimized for compact size from the original.
+“Valid HTML5 Intrinsic Web Badge” is licensed under <a rel="nofollow"
+class="external text" href="https://creativecommons.org/licenses/by/3.0/">CC
+BY</a> by Derek Lewis, Certified Web Developer, and high-marking college student
+of Artificial Intelligence based in-and-around the Magic City of Miami, Florida.
+
+---
+
+### Valid CSS3 Intrinsic Web Badge
+
+```html
+<p>
+  <a href="https://validator.w3.org/check?uri=referer">
+    <img src="valid-html5.svg" alt="Valid CSS3!" height="21" width="81" />
+  </a>
+</p>
+```
+
+<p>
+  <a href="https://validator.w3.org/check?uri=referer">
+    <img src="valid-css3.svg" alt="Valid CSS3!" height="21" width="81" />
+  </a>
+</p>
```

Comment:
```suggestion
```html
<p>
  <a href="https://validator.w3.org/check?uri=referer">
    <img src="https://open.inf.is/assets/img/svg/valid-css3.svg?sanitize=true" alt="Valid CSS3!" height="21" width="81" />
  </a>
</p>
```

<p>
  <a href="https://validator.w3.org/check?uri=referer">
    <img src="https://open.inf.is/assets/img/svg/valid-css3.svg?sanitize=true" alt="Valid CSS3!" height="21" width="81" />
  </a>
</p>
```